include/openssl/cmp.h.in

```diff
@@ -36,7 +36,9 @@ use OpenSSL::stackhash qw(generate_stack_macros);
 extern "C" {
 #  endif
 
-#  define OSSL_CMP_PVNO 2
+#  define OSSL_CMP_PVNO 2 /* still the default */
```

Comment:
_DEFAULT might of been better in hindsight..

Reply:
Unfortunately that define is already present in current API.